docs/_macros/version_cleanup.md

```diff
@@ -0,0 +1,26 @@
+---
+layout: default
+class: Builder
+title: version_cleanup ';' VERSION-STRING
+summary: Cleanup a potential maven version to make it match an OSGi Version syntax.
+---
+
+
+```java
```

Comment:
This place holder code should be removed and some words put here to describe what the macro does to the version argument.